modules/virt-downloading-virtctl-client-online.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// virt/install/virt-installing-virtctl.adoc
+
+[id="virt-downloading-virtctl-client-online_{context}"]
```

Comment:
We don't have this mechanism anymore. We have only 1 mechanism which allows users to download virtctl from the running cluster.